src/Server.php

```diff
@@ -139,6 +139,19 @@ public function handleRequest(ConnectionInterface $conn, Request $request)
             }
         }
 
+        if ($request->hasHeader('Content-Length')) {
```

Comment:
Have you checked this wrt to chunked transfer encoding?

In particular wrt validating requests that contain both chunked transfer encoding AND a `Content-Length`?